microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,92 @@
 
-Microsoft recommends that you use [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] whenever possible for the following reasons.  
+Mechanisms for browser detection with device (client or server) and identifier:
 
-*   Improve code maintainability.  
-*   Reduce code fragility.  
-*   Reduce code breakage from changes to the User-Agent string.  
-    
-When [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable and you must use user agent detection, use the following format to detect Microsoft Edge user agent on Windows and Android.  
+| Mechanism | Device and Identifier |  
+|:--- |:--- | 
+| **Feature Detection** \(recommended\) | Client-side: `navigator.userAgentData.brands` JavaScript method |
+| **User-Agent Client Hints** \(recommended\) | Server-side: `Sec-CH-UA` HTTPS header |
+| | Client-side: `navigator.userAgentData` JavaScript method |  
+| **User-Agent String** \(legacy\) | Server-side:`User-Agent` HTTPS header | 
+| | Client-side: `navigator.userAgent` JavaScript method |  
 
-## User-Agent Client Hints  
+This article describes the methods Microsoft Edge supports for retrieving user agent information.
 
-Starting in Microsoft Edge version 90, access browser information in a cleaner, more privacy-preserving way.  
+## Feature detection
 
-### User-Agent Client Hints HTTPS Header  
+Microsoft recommends detecting if a feature is supported in your browser. For more information, see [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection].
+
+If you must detect browsers, Microsoft recommends using client hints as follows.
+
+## User-Agent Client Hints
+
+Microsoft Edge supports User-Agent Client Hints starting with version 90.
+
+User-Agent Client Hints is a cleaner, more privacy-preserving way to access browser information such as browser name, version number, platform, and more. Soon the User-Agent string will be frozen and deprecated by various browsers. For example, the Chrome Platform Status site describes the change in [Feature: Reduce User-Agent string information][https://www.chromestatus.com/feature/5704553745874944].
+
+Employ User-Agent Client Hints when you want to:
+- Determine if new browser activity is from the expected user.
+- Customize hints or instructions if the user is new to this site.
+
+Do not use User-Agent Client Hints to:
+- Block *unsupported* browsers.
+- Restrict access to features on your site.
+
+For more information, navigate to the W3C Community Draft Report at [User-Agent Client Hints][https://wicg.github.io/ua-client-hints/].
+
+### User-Agent Client Hints HTTPS header
+
+When a browser requests a web page from a server, it sends an HTTPS client header request. The server replies with a specific set of HTTPS response header data. User-Agent Client Hints automatically return the brand and mobile attributes. It can easily include more data.

```

Comment:
```suggestion
When a browser requests a web page from a server, it sends an HTTPS client header request. The server replies with a specific set of HTTPS response header data. User-Agent Client Hints automatically returns the brand and mobile attributes. It can easily include more data.

```

Reply:
Replaced the whole thing with better wording. Thanks for checking!